src/applications/vaos/containers/ClinicChoicePage.jsx

```diff
@@ -79,14 +79,19 @@ export class ClinicChoicePage extends React.Component {
     }
 
     const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
-
+    let pageTitle;
+    if (schema.properties.clinicId.enum.length === 2) {
+      pageTitle = `Make a ${typeOfCareLabel} appointment at your last clinic`;
+      document.title = `${pageTitle} | Veterans Affairs`;
```

Comment:
Technically we shouldn't set `document.title` in the render method, because React doesn't expect DOM changes here. This one's kinda tricky, though. I think I would make a function that took `schema` as a parameter and returned the page title, then called that in here to use in the `h1` and is also called in `componentDidUpdate` to get the title for `document.title`.

So, something like:

```js
function getPageTitle(schema, typeOfCare) {
  const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
  let pageTitle = 'Clinic choice';
  if (schema?.properties.clinicId.enum.length === 2) {
    pageTitle = `Make a ${typeOfCareLabel} appointment at your last clinic`;
  } else if (schema?.properties.clinicId.enum.length > 2) {
    pageTitle = `Select your VA clinic for your ${typeOfCareLabel} appointment`;
  }
  return pageTitle;
}
```

Reply:
@jbalboni, the getPageTitle function worked out well for on ClinicChoicePage.jsx but for some reason it failed the unit test with the following msg:

"VAOS <ClinicChoicePage> document title to match h1 text in single clinic:
 AssertionError: expected '' to include 'Make a primary care appointment at your last clinic' 
at Context.contain (src/applications/vaos/tests/containers/ClinicChoicePage.unit.spec.jsx:83:28)

Reply:
@jbalboni, would you take a look at the failed unit test? I pushed all three updated files to the same PR. Thanks.

Reply:
I pushed up a fix for the unit test. The issue is that, you're setting the page title in `componentDidUpdate`, so we need to make sure that method is run. The way the React lifecycle works is that `componentDidMount` runs the first time a component is rendered, then `componentDidUpdate` runs after any future change. So in the test I used our test library's `setProps` method to make the component update and set the props so that the conditional your code is in will pass.

Reply:
Thank Jeff for explaining. I'll be sure to pull down the unit test to see how to use setProps.